lib/utils.pm

```diff
@@ -418,7 +418,20 @@ sub ensure_unlocked_desktop {
                 type_string "$username";
                 save_screenshot;
             }
-            send_key 'ret';
+            # On gnome, user may not be selected and using 'ret' is not enough in this case
+            if (check_var('DESKTOP', 'gnome') && (is_sle('15+') || is_leap('15.0+') || is_tumbleweed)) {
```

Comment:
We get same code in `handle_login`, this code is used everywhere, so we have to be careful by changing it too.

Reply:
We could at least extract same code to separate method. But I would like to see verification runs for SLE with gnome and textmode and different desktops for any openSUSE distri. This is used in too many places and is pretty fragile to such changes =(

Reply:
I do not have access to SLE tests. Could you run the tests for me?

Reply:
Sure, I can support you with that, but let's solve point above and get verification runs for openSUSE first.